Modules/Setup

```diff
@@ -156,6 +156,7 @@ PYTHONPATH=$(COREPYTHONPATH)
 #math mathmodule.c
 #mmap mmapmodule.c
 #select selectmodule.c
+#_lazydebug _imp.c
```

Comment:
what's this?

Reply:
Oops, that is I did to mimic the implementation of `_crypt`, but I found it doesn't work. Let me remove it.